variables.tf

```diff
@@ -376,4 +376,16 @@ variable "tcp_keepalive_probes" {
   description = "Sets net.ipv4.tcp_keepalive_probes (number), currently only supported in ECS."
   type        = number
   default     = 2
-}
\ No newline at end of file
+}
+
+variable "datadog_key_secret_name" {
```

Comment:
new variables need to be added to README. Please add in the variable's description that this needs to be created in AWS secrets manager.

Reply:
Added them in README. Thanks